edgelet/edgelet-kube/src/settings.rs

```diff
@@ -24,6 +24,7 @@ pub struct Settings {
     device_id: Option<String>,
     device_hub_selector: String,
     proxy: ProxySettings,
+    has_cluster_rbac: Option<bool>,
```

Comment:
We can make default `true` via serde features. This way we don't need the field to be Option. https://serde.rs/attr-default.html

Reply:
I was looking for that, didn't find it in Rust online docs, thank you!